### PR TITLE
Hot loader styles

### PR DESCRIPTION
**Pull request by olizilla**
Simplify and consolidate the loader styles. There is an easter egg for the subs loader, but I see it so briefly that I can't screenshot it now...

screenshot 2016-05-19 11 16 41

### CODE REVIEW

**Comment by richsilv** [comment]
:eyeglasses:

**Comment by olizilla** [comment]
Fixes #27

File: imports/ui/pages/layout.jsx [hunk 1]
```diff
@@ -28,11 +28,11 @@ const Layout = React.createClass({
       <div className='full-height'>
         <SubscribeProjects />
         <Navbar />
-        <div className='content m-b-1'>
+        <Loader top='32px' left='calc(100% - 35px)' />
```
Comment (richsilv) [inline imports/ui/pages/layout.jsx]: Don't really understand the props here.  If the parent is indeed full-height, won't the loader be centered anyway?

Comment (richsilv) [inline imports/ui/pages/layout.jsx]: Ignore that, I'm looking in the wrong place!  It's NICE.

**Comment by richsilv** [comment]
Looks really good.  :+1:

**Pull request merged**